ARCH/vpr_arch/k4_N8_tileable_reset_softadder_register_scan_chain_nonLR_caravel_io_skywater130nm.xml

```diff
@@ -350,7 +350,7 @@ Authors: Xifan Tang
             <direct name="ff[1:1]-DI" input="ff[0:0].Q" output="ff[1:1].DI"/>
             <direct name="iopad-sc_out" input="ff[1:1].Q" output="iopad.sc_out"/>
             <complete name="complete1" input="iopad.reset" output="ff[1:0].reset"/>
-            <mux name="mux1" input="iopad.f2a_i ff[0:0].Q iopad.clk" output="pad.outpad">
+            <mux name="mux1" input="iopad.f2a_i ff[0:0].Q" output="pad.outpad">
```

Comment:
On behalf of Tarachand, to remove the "iopad.clk" input, until "physical" mode honors the "disabled_in_pack=true" (if I understand correctly).